lib/actions/notes.coffee

```diff
@@ -30,4 +30,4 @@ exports.set =
 		if _.isEmpty(params.note)
 			return done(new Error('Missing note content'))
 
-		resin.models.device.note(options.device, params.note, done)
+		resin.models.device.note(options.device, params.note).nodeify(done)
```

Comment:
Can you edit the `options.device` description and the command example to use uuids as the other device commands?